modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/client/bundle/create/BundleUploadDistroFileStep.java

```diff
@@ -297,6 +301,24 @@ public void onFailure(Throwable caught) {
             });
     }
 
+    private BundleNotFoundException unpackBundleNotFoundException(Throwable caught) {
+        if (caught instanceof RuntimeException) {
+            String message = caught.getMessage();
+            int patternStart = message.indexOf(BUNDLE_NOT_FOUND_EXCEPTION_PATTERN_START);
+            if (patternStart > 0) {
```

Comment:
Please use -1 to indicate something wasn't found, as now it could have been found from index 0